client-src/index.js

```diff
@@ -236,6 +239,18 @@ const onSocketMessage = {
       );
     }
 
+    if (isProgressSupported()) {
```

Comment:
Let's move `defineProgressElement` after this check to avoid extra checks and have `customElements.define("wds-progress", WebpackDevServerProgress);` only in this function